documentation/docs/02-template-syntax/07-special-elements.md

```diff
@@ -200,7 +200,6 @@ If `this` is the name of a [void element](https://developer.mozilla.org/en-US/do
 <script>
 	let tag = 'div';
 
-	/** @type {(e: MouseEvent) => void} */
```

Comment:
why does this have to be removed?

Reply:
Prettier is erroring with this line for some reason. Something deeper in the code itself, but 2 hours of debugging didn't reveal the main issue